cumulusci/tasks/salesforce/ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,36 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ActivateProcessBuilderProcesses(BaseSalesforceApiTask):
+    """
+    Create a Salesforce Activation Process Builder which takes in a list as options and
+    runs flows only on those listed DeveloperName's.
+    """
+
+    def _run_task(self):
+        self.logger.info(
+            f'Allowing only rows with the following DeveloperNames {self.options.get("developer-names")}'
```

Comment:
I'd rephrase this log to express what it's doing in Salesforce-y terms. Maybe something like "Activating the following Flows:".

Reply:
I Like that!